src/main/java/net/fabricmc/tinyremapper/api/MemberHeader.java

```diff
@@ -0,0 +1,44 @@
+package net.fabricmc.tinyremapper.api;
+
+import java.util.Objects;
+
+public class MemberHeader {
```

Comment:
Is there any specific reason to not make this an interface inherited by `MemberInstance`?